src/features/transfer-ownership/FormSteps/Complete.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react';
+
+import { StepProps } from './types';
+
+import { Wizard } from '@zero-tech/zui/components/Wizard';
+
+export const Complete: FC<StepProps> = ({ onClose }) => (
+	<>
+		<Wizard.Confirmation
+			onClickPrimaryButton={onClose}
+			primaryButtonText={'Finish'}
+			isPrimaryButtonActive
+			message={'Transfer Successful'}
+		/>
+	</>
```

Comment:
You won't need to wrap this in a Fragment, since `Wizard.Confirmation` doesn't have any siblings.

Reply:
missed these in refactoring! thanks for spotting